src/mds/MDSRank.cc

```diff
@@ -4111,6 +4111,7 @@ const char** MDSRankDispatcher::get_tracked_conf_keys() const
 void MDSRankDispatcher::handle_conf_change(const ConfigProxy& conf, const std::set<std::string>& changed)
 {
   // XXX with or without mds_lock!
+  dout(2) << __func__ << ": " << changed << dendl;
```

Comment:
Just a query: Is there any doc explaining about the log levels. I mean which level/number should we use for what kind of log?

Reply:
I don't think there is anything written down. Roughly:

1 is on by default, so very selectively use it.
2-5 is useful but infrequent.
5-10 is for basic debugging of MDS.
11-20 is for the firehose of information.